lib/sensu/extensions/occurrences.rb

```diff
@@ -21,11 +38,11 @@ def description
       # @return [Array] containing filter output and status.
       def event_filtered?(event)
         check = event[:check]
-        occurrences = check[:occurrences] || 1
-        refresh = check[:refresh] || 1800
+        occurrences = str2int(check[:occurrences]) || 1
+        refresh = str2int(check[:refresh]) || 1800
```

Comment:
any reason to not just `check[:occurrences].to_i` seems like reinventing the wheel to me.